HISTORY.md

```diff
@@ -1,14 +1,8 @@
 ### ensmallen ?.??.?: "???"
 ###### ????-??-??
- * Modify matrix initialisation to take into account
-   default element zeroing in Armadillo 10.5
-   ([#305](https://github.com/mlpack/ensmallen/pull/305)).
 
- * Disable building the tests by default for faster installation
-   ([#303](https://github.com/mlpack/ensmallen/pull/303)).
-
- * Improved installation and compilation instructions
-   ([#300](https://github.com/mlpack/ensmallen/pull/300)).
+### ensmallen 2.17.0: "25 Din Mein Paisa Double"
+###### 2021-05-07
```

Comment:
Actually, @jonpsy, I just realized, this date is backwards with the format we have been using in other places---this says May 7, but it should be `2021-07-05` (e.g. July 5).  Did you change anything in the script having to do with the date?

Reply:
Ah my bad, didn't change the script. I just wrote in YYYY/DD/MM format by mistake in HISTORY.md